tools/@aws-cdk/prlint/action.yml

```diff
@@ -1,5 +1,5 @@
 name: Pull Request Linter
 description: Execute validation rules on GitHub Pull Requests
 runs:
-  using: node12
+  using: node14
```

Comment:
Looks like `node14` is not supported here?

Reply:
Yep, you are correct. I'll revert this really quick. https://github.com/actions/runner/issues/772